_search-plugins/search-pipelines/rerank-processor.md

```diff
@@ -110,9 +117,166 @@ POST /_search?search_pipeline=rerank_pipeline
 
 The `query_context` object contains the following fields. 
 
-Field name  | Description
-:--- | :---  
-`query_text` | The natural language text of the question that you want to use to rerank the search results. Either `query_text` or `query_text_path` (not both) is required.
-`query_text_path` | The full JSON path to the text of the question that you want to use to rerank the search results. Either `query_text` or `query_text_path` (not both) is required. The maximum number of characters in the path is `1000`.
+Field name | Required/Optional | Description
+:--- | :--- | :---  
+`query_text` | Required (see below)| The natural language text of the question that you want to use to rerank the search results. 
+`query_text_path` | Required (see below) | The full JSON path to the text of the question that you want to use to rerank the search results. The maximum number of characters in the path is `1000`.
+
+You are required to select either `query_text` or `query_text_path`, but not both simultaneously. Please ensure that only one option is chosen to avoid conflicts
```

Comment:
```suggestion
You are required to select either `query_text` or `query_text_path` but cannot select both simultaneously. Please ensure that only one option is chosen in order to avoid conflicts.
```